.github/workflows/buildBackupParamaterStore.yaml

```
@@ -0,0 +1,27 @@
+name: Build Backup Parameter Store
+on:
+  push:
+    paths:
+      - 'backup-parameter-store/**'
+jobs:
+  backup-parameter-store:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: aws-actions/configure-aws-credentials@v1
```

Can we use the latest versions?

```suggestion
     - uses: actions/checkout@v4
     - uses: aws-actions/configure-aws-credentials@v4
```

See:
* https://github.com/actions/checkout/releases
* https://github.com/aws-actions/configure-aws-credentials/releases